tasks/main.yml

```diff
@@ -19,7 +19,7 @@
       - nss
       - curl
       - libcurl
```

Comment:
We probably should clean up post-build anything we no longer need to make it a more palatable role.
Also, why do we need both wget and curl?